zerver/migrations/0082_emoji_reactions_codepoint.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-05-01 05:26
+from __future__ import unicode_literals
+
+from os.path import abspath, dirname, join
+import ujson
+
+from django.conf import settings
+from django.db import migrations, models
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+ZULIP_PATH = dirname(settings.STATIC_ROOT)
+path_to_unified_reactions = join(ZULIP_PATH, "tools", "setup", "emoji", "unified_reactions.json")
+unified_reactions = ujson.load(open(path_to_unified_reactions))
```

Comment:
This code should be moved into the `copy_emoji_name` function.  

Otherwise, this will slow commands like `manage.py showmigrations` that need to import all the migrations but don't need to access these data.